bin/all_sky_search/pycbc_sngls_statmap

```diff
@@ -128,19 +122,18 @@ for key in all_trigs.seg.keys():
 f['segments/foreground_veto/start'] = numpy.array([0])
 f['segments/foreground_veto/end'] = numpy.array([0])
 for k in all_trigs.data:
-    f['foreground/' + k] = all_trigs.data[k][fore_locs]
-    f['background/' + k] = all_trigs.data[k][back_locs]
-    f['background_exc/' + k] = all_trigs.data[k][back_locs]
-
+    f['foreground/' + k] = all_trigs.data[k]
+    f['background/' + k] = all_trigs.data[k]
 
 logging.info("Estimating FAN from background statistic values")
 # Ranking statistic of foreground and background
-fore_stat = all_trigs.stat[fore_locs]
-back_stat = all_trigs.stat[back_locs]
-back_stat_exc = all_trigs.stat[back_locs]
+fore_stat = back_stat = all_trigs.stat
+# Need to make a copy as this will have items removed from it, and dont want
+# to overwrite the original
+back_stat_exc = copy.deepcopy(back_stat)
 
-bkg_dec_facs = all_trigs.decimation_factor[back_locs]
-bkg_exc_dec_facs = all_trigs.decimation_factor[back_locs]
+bkg_dec_facs = all_trigs.decimation_factor
+bkg_exc_dec_facs = all_trigs.decimation_factor
```

Comment:
probably clearer to do this via deepcopy as well - ie if you've used it once for a specific reason which requires a comment to explain, you can use it again

Reply:
Done